source/_components/transmission.markdown

```diff
@@ -0,0 +1,115 @@
+---
+layout: page
+title: "Transmission"
+description: "Instructions on how to integrate Transmission within Home Assistant."
+date: 2018-12-12 22:44
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: transmission.png
+ha_category: Downloading
+ha_release: 0.85
+ha_iot_class: "Local Polling"
+---
+
+The `transmission` component allows you to monitor your downloads with [Transmission](http://www.transmissionbt.com/) from within Home Assistant and setup automation based on the information.
+
+## {% linkable_title Setup %}
+
+To use the monitoring, your transmission client needs to allow remote access. If you are running the graphical transmission client (transmission-gtk) go to **Edit** -> **Preferences** and choose the tab **Remote**. Check **Allow remote access**, enter your username and your password, and uncheck the network restriction as needed.
+
+<p class='img'>
+  <img src='{{site_root}}/images/components/transmission/transmission_perf.png' />
+</p>
+
+If everything is setup correctly, the details will show up in the frontend.
+
+<p class='img'>
+  <img src='{{site_root}}/images/components/transmission/transmission.png' />
+</p>
+
+## {% linkable_title Configuration %}
+
+To enable this sensor, add the following lines to your `configuration.yaml`:
+
+```yaml
+transmission:
+  host: 192.168.1.1
+  username: user
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.